### DISCUSSION TITLE
Is there a way to add a border/stroke around the modal overlay cutout?

### DISCUSSION THREAD

**Discussion by tettoffensive** [shipshapecode/shepherd#2470] [Q&A]
It looks like in the implementation, the "cutout" around attached element with useModalOverlay is the same path as the entire modal overlay. So I can't figure out how to target just the cutout to add a stroke or border along just this portion of the path.

Is something like this supported? Or might there be a way I could achieve this?

**Comment by RobbieTheWagner** [comment]
@tettoffensive that is correct, we use one SVG with a hole cut out of it. Can you describe your use case more?

**Reply by tettoffensive** [reply]
The design spec has an 8px blue outline around the cutout transparent area. I'm not at a computer at the moment but if that's not clear I can share an image later

**Reply by RobbieTheWagner** [reply]
I understand what you're saying, I just don't know what the benefit is? We know the coordinates of the cutout, so it could be theoretically possible to add some kind of function to generate an SVG outline, but I don't think this is something many people would need.

**Reply by tettoffensive** [reply]
I'm assuming the designer designed it that way to help draw the users attention to the cutout area. Perhaps they thought just the cutout and shaded area wasn't attention grabbing enough or perhaps it was just an aesthetic decision. 

I'm implementing the design as close as possible. But based on your response it sounds like I might suggest to them that we drop that as it may be more effort than worth.

**Reply by RobbieTheWagner** [reply]
Yes, I think removing that would probably be the easiest option.
